### ISSUE TITLE
H1 tag type error

### ISSUE THREAD

**Issue by linusnorton** [govuk-react/govuk-react#1138]
**Describe the bug**
I get a typescript compilation error when using H1 tags:

```
TS2559: Type '{ children: string; }' has no properties in common with type 'IntrinsicAttributes & HeadingOwnProps & WithWhiteSpaceProps'
```

**To Reproduce**
```
import { H1 } from 'govuk-react';

<H1>Text</H1>
```

**Expected behavior**
No errors

**Screenshots**
N/A

**Comment by penx** [comment]
AFAIK this is an issue with the React 18 typings removing `children` being there by default

https://stackoverflow.com/questions/71788254/react-18-typescript-children-fc

We'll need to address this in govuk-react by adding the children prop in manually for all components and test these against the React 18 types.

Would you like to submit a pull request for this?

**Comment by linusnorton** [comment]
Resolved in 0.10.4
